src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/RemoveEmptyFinalizers.Fixer.cs

```diff
@@ -31,7 +30,7 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
             }
 
             string title = MicrosoftCodeQualityAnalyzersResources.RemoveEmptyFinalizers;
-            context.RegisterCodeFix(new MyCodeAction(title,
+            context.RegisterCodeFix(CodeAction.Create(title,
                              async ct => await RemoveFinalizerAsync(context.Document, node, ct).ConfigureAwait(false),
                              equivalenceKey: title),
```

Comment:
We probably need to make a pass and make equivalenceKey consistent across both our repos to have the non-localized string instead of passing in the title. Not required for this PR.

Reply:
Possibly also write an RSxxxx meta-analyzer (and maybe a fixer) to flag non-constant string being passed to the `equivalenceKey` parameter to `CodeAction.Create` invocations and switch to using the nameof pattern that we have been using recently.

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/5431